game/match.py

```diff
@@ -41,5 +37,14 @@ def who_won(self):
                 return player
         return None
 
+    def retrieve_board(self):
+        return self._board.copy()
+
+    def _players(self):
+        return [self._player_one, self._player_two]
+
+    def get_players(self):
```

Comment:
I don't see why you need both of these. Just define one version (with tuples or arrays).